dwave/cloud/api/client.py

```diff
@@ -315,7 +315,10 @@ def configure_cache(self, cache: CacheConfig) -> None:
             logger.debug("[%s] cache disabled.", type(self).__name__)
             return
 
-        self._maxage = config['maxage']
+        if not isinstance(_maxage := config.get('maxage'), int) or _maxage < 0:
+            raise ValueError("A non-negative integer required for 'maxage'.")
+        self._maxage = _maxage
```

Comment:
```python
       _maxage = config.get('maxage')
        if not isinstance(_maxage, int) or _maxage < 0:
            raise ValueError("A non-negative integer required for 'maxage'.")
        self._maxage = _maxage
```
Maybe this is an aesthetic preference but I found it hard to find where `_maxage` for instance came from when it's assigned in the if branch.

Reply:
OK, I can update this in a follow-up PR (that I'm already working off this branch, and where I changed this exact line) to simplify conflicts on rebase.

Reply:
Done in https://github.com/dwavesystems/dwave-cloud-client/pull/651.